env/Dockerfile

```diff
@@ -134,6 +135,7 @@ RUN chown -R ${_USER}:${_USER} ${_HOME_DIRECTORY}
 #BASH COMPLETION
 USER magento
 RUN echo "source /etc/bash_completion" >> ${_HOME_DIRECTORY}/.bashrc
+RUN echo 'alias m="php bin/magento"' >> ~/.bash_aliases
```

Comment:
This one is quite generic and may interfere with other aliases/tools used inside the container.

Starting from v2.0 /home folder is persistent (has its own volume) so you can add an alias once and it will be preserved between rebuilds.